### ISSUE TITLE
Cannot Login (Possibly due to 2FA?)

### ISSUE THREAD

**Issue by danilofilippo** [web-pal/chronos-timetracker#148]
## Expected Behavior

After email/password input, a 2-factor-authentication prompts for a code to continue with login

## Actual Behavior

After inputing the email and password, it keeps loading infinitely. No message on the Auth debug console, not even a timeout of sorts. Restarting the application/clearing cache does not solve the problem.

## Steps to Reproduce the Problem

  1. Have a jira account with 2-factor-authentication configured
  2. Input the JIRA host, click continue
  3. Input email/password, click continue

Obs.: I have no clue wheter the 2FA is actually related to the problem, it's just what i imagine it to be.

## Specifications

  - Chronos-timetracker version: 2.8.6
  - OS: Windows 10 x64

**Comment by VladimirPal** [comment]
@danilofilippo  Hi!
Could you try please 3.0.0-beta, it should be fixed

**Comment by danilofilippo** [comment]
Hi @VladimirPal, thanks for the reply! 

I just tried with version 3.0.0-beta (from [here](https://github.com/web-pal/chronos-timetracker/releases/tag/v3.0.0-beta)), but the behaviour is still the same.

The auth debug console still shows no message. I then tried to download the debug version and it seems that no request is sent after I input my email/password, which I found kind of weird (but I never developed anything based on electron, so I'm not really sure that is supposed to show).

Feel free to ask for more tests at my end if needed. (Is there a log file somewhere I can attach?)
